### PR TITLE
Translations batch 07 2021

### PR DESCRIPTION
**Pull request by rsanheim**
[Story ch4103](https://app.clubhouse.io/simpledotorg/story/4103/translations-batch-july-2021)
 
## New Languages

* [si_LK - Sinhala - Sri Lanka](https://lh.2xlibre.net/locale/si_LK/)
* [ta_LK - Tamil - Sri Lanka](https://lh.2xlibre.net/locale/ta_LK/)

## Checklist 
* [x] https://github.com/simpledotorg/simple-server/pull/2672
* [x] https://github.com/simpledotorg/simple-server/pull/2680
* [x] https://github.com/simpledotorg/simple-server/pull/2682
* [x] Make any necessary submissions to WhatsApp
* [x] Verify CI
* [ ] Verify in the app
* [ ] Deploy

### Closed and not merged (just changes a top level key)
* [x] https://github.com/simpledotorg/simple-server/pull/2670
* [x] https://github.com/simpledotorg/simple-server/pull/2673
* [x] https://github.com/simpledotorg/simple-server/pull/2674
* [x] https://github.com/simpledotorg/simple-server/pull/2675
* [x] https://github.com/simpledotorg/simple-server/pull/2676
* [x] https://github.com/simpledotorg/simple-server/pull/2677
* [x] https://github.com/simpledotorg/simple-server/pull/2678
* [x] https://github.com/simpledotorg/simple-server/pull/2679
* [x] https://github.com/simpledotorg/simple-server/pull/2681
* [x] https://github.com/simpledotorg/simple-server/pull/2671

### CODE REVIEW

**Review by harimohanraj89 — CHANGES_REQUESTED** [review]
Neato. Since this includes a couple new languages, we'll have to add them (Sinhala and Sri Lankan Tamil) to our supported locales in `config/application.rb`.

**Comment by rsanheim** [comment]
> Neato. Since this includes a couple new languages, we'll have to add them (Sinhala and Sri Lankan Tamil) to our supported locales in `config/application.rb`.

Yup added.  I have some questions for ya in the linked clubhouse story as well.

File: config/application.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Application < Rails::Application
 
     # Locale configuration
     config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**", "*.{rb,yml}")]
-    config.i18n.available_locales = %w[en mr-IN pa-Guru-IN bn-BD kn-IN en_BD en_IN en_ET en-IND bn-IN hi-IN ta-IN te-IN am-ET om-ET sid-ET so-ET ti-ET]
+    config.i18n.available_locales = %w[en mr-IN pa-Guru-IN bn-BD kn-IN en_BD en_IN en_ET en-IND bn-IN hi-IN ta-IN te-IN am-ET om-ET si_LK sid-ET so-ET ta_LK ti-ET]
```
Comment (harimohanraj89) [inline config/application.rb:44]: Ooooh fun decision time. Hyphens or underscores?
* Hyphens: Consistent with the other languages but constantly have to fight Transifex (the language mapping config in Transifex doesn't apply to the YML keys)
* Underscores: Inconsistent with current state but no Transifex friction

Is there a standard we should follow? Or are the locales we configure here arbitrary?

**Review by harimohanraj89 — APPROVED** [review]
Code looks good. Whatsapprovals pending of course.

I have one last question about our locale conventions, I wonder if we should bite the bullet and start switching 🤔

File: config/application.rb (hunk 1 above)
Comment (rsanheim) [reply config/application.rb:44]: I matched what Transifex submitted for the new languages here...is there any reason _not_ to do that, other than that inconsistency with the other locales?

Comment (harimohanraj89) [reply config/application.rb:44]: Not that I know of 🤔

**Comment by rsanheim** [comment]
> I have one last question about our locale conventions, I wonder if we should bite the bullet and start switching 🤔

Yeah I think so, anything to make the transifex process easier seems worth while to me.

**Pull request merged**